metis_client/helpers.py

```diff
@@ -126,11 +126,11 @@ def http_to_metis_error_map(status: int) -> Type[MetisError]:
 def metis_error_to_raise(dto: MetisErrorDTO):
     "Raise MetisErrorDTO"
     if isinstance(dto["error"], dict):
-        msg = str(dto["error"]["message"])
+        msg = str(dto["error"].get("message", "Unexpected server error"))
```

Comment:
Why server should return dict instead of string, but with different unknown format? Ok, but strange.

Reply:
KeyError occurs under certain erroneous server responses